src/createServer_AIO.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+// *** NOTE: ***
+// - In this file server doesnt split by components and modules
+// |-- so you can see All In One(AIO) place:
```

Comment:
what's the purpose of this file? you have all the functionality here (both for users and expenses), and you also have separate controllers with these functions, right?
if you have the splitted controllers with this functionality, let's delete this one